packages/wast-parser/src/tokenizer.js

```diff
@@ -112,6 +112,32 @@ function tokenize(input: string) {
 
   const tokens = [];
 
+  /**
+   * Can be used to look at the next character(s).
+   *
+   * The default behavior `lookahead()` simply returns the next character without consuming it.
+   *
+   * @param int length How many characters to query. Default = 1
+   * @param int offset How many characters to skip forward from current one. Default = 1
+   *
+   */
+  function lookahead(length = 1, offset = 1) {
+    return input.substring(current + offset, current + offset + length);
+  }
+
+  /**
+   * Can be used to look at the last few character(s).
+   *
+   * The default behavior `lookback()` simply returns the last character.
+   *
+   * @param int length How many characters to query. Default = 1
+   * @param int offset How many characters to skip back from current one. Default = 1
+   *
+   */
+  function lookback(length = 1, offset = 1) {
```

Comment:
Could we rename it to `lookbehind`? It sounds more common to me.

Reply:
Yeah that sounds better :+1: